woocommerce/woocommerce-android
# Product creation using package photo: OCR library integration

This PR integrates the ML Kit library for text recognition, which is used to display keywords for product title and description generation (next PR). Also, the loading and error state UI will be added in the next PR.

This PR also updates the README to reflect that we are using ML Kit library.

**To test:**
1. Go to Products
2. Tap on + FAB
3. Select the AI option
4. Tap on Use package photo button
5. Use the media library to select a picture of a product with some text
6. Notice the image is loaded and a list of keywords from the image is displayed
7. Try using a different image

<img src="https://github.com/woocommerce/woocommerce-android/assets/1522856/433d9a86-cf5e-4c2f-a763-856ac97e224c" width="500" />

*Note:* Please merge #10003 first.
